assets/js/lib/model/clusters.js

```diff
@@ -1,8 +1,24 @@
+// Cluster types
 export const HANA_SCALE_UP = 'hana_scale_up';
 export const HANA_SCALE_OUT = 'hana_scale_out';
 export const ASCS_ERS = 'ascs_ers';
 
-export const clusterTypes = [HANA_SCALE_UP, HANA_SCALE_OUT, ASCS_ERS];
+// Hana scale up scenarios
+export const HANA_SCALE_UP_PERFORMANCE_SCENARIO = 'performance_optimized';
```

Comment:
Let's remove the `HANA_SCALE_UP` prefix from here.
Actually, Scale out are performance optimized as well